Add arrow props and followCuror for Tooltip and Popover (#2508)
rsuite/rsuite: docs/pages/components/popover/fragments/follow-cursor.md

<!--start-code-->

```js
const instance = (
  <Whisper
    followCursor
    speaker={

>       <Popover>This is a ToolTip for simple text hints. It can replace the title property</Popover>

Reviewer: ```suggestion
      <Popover>This is a Popover that follow cursor</Popover>
```
